frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -147,6 +150,17 @@ const HostSummary = ({
             id={"disk-space-tooltip"}
           />
         </div>
+        {typeof diskEncryption?.enabled === "boolean" &&
+        diskEncryption?.tooltip ? (
+          <div className="info-flex__item info-flex__item--title">
+            <span className="info-flex__header">Disk encryption</span>
+            <TooltipWrapper tipContent={diskEncryption.tooltip}>
+              {diskEncryption.enabled ? "On" : "Off"}
+            </TooltipWrapper>
+          </div>
+        ) : (
+          <></>
+        )}
```

Comment:
Is the type check necessary? You can force the value to a boolean with a `!!` operator: `{!!diskEncryption.enabled ? "On" : "Off"}`

Since you want to make sure both values are there, you'd still need to do `diskEncryption?.enabled && diskEncryption?.tooltip` but it would allow you to remove the type check.

Reply:
@lukeheath I'm not sure if I follow.

The specs have it if `diskEncryption.enabled` key set to `null` (which is only possible for Linux), we do not render the Disk encryption information at all since we can't determine if it's on or off. How might I hide the entire div without doing a type check?

Reply:
@RachelElysia Ah got it, I wasn't think of that requirement. This makes sense then, thanks.